src/main/java/com/atlassian/bitbucket/jenkins/internal/model/BitbucketRepository.java

```diff
@@ -39,6 +39,16 @@ public BitbucketRepository(
         }
     }
 
+    public BitbucketRepository(List<BitbucketNamedLink> cloneUrls, String name, BitbucketProject project,
```

Comment:
/nit ordering of parameter same as other constructors. This will make it easier to identify the difference.